.github/workflows/php-quality.yml

```
@@ -124,3 +124,37 @@ jobs:
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '8.2'
+          coverage: none
+          tools: composer
+
+      - name: Get Composer cache directory
+        id: composer-cache
+        run: echo "dir=$(composer config cache-files-dir)" >> $GITHUB_OUTPUT
+
+      - name: Cache Composer dependencies
+        uses: actions/cache@v3
```

actionlint (1.7.12)

[error] the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)